core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -40,6 +40,12 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-aws-ipi-byo-route53:
     platform: aws
+  cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts:
+    platform: aws
+  cucushift-installer-rehearse-aws-ipi-shared-vpc-phz:
+    platform: aws
+  cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts:
```

Comment:
Would you mind to sort them in alphanumeric order ?

Reply:
done